app/models/vote.rb

```diff
@@ -0,0 +1,5 @@
+class Vote < ApplicationRecord
+  # is it correct assotiations for my task? (has_many in user- and micropost models also)
```

Comment:
Yes, it looks correct. I'd recommend to add validations, because a vote should have user and micropost, there shouldn't be a vote with nil reference

Reply:
There's `null: false` in the migration, but also makes sense to validate fields in the model